### ISSUE TITLE
Consistently Showing Cycle Route Names 

### ISSUE THREAD

**Issue by paultroop** [cyclosm/cyclosm-cartocss-style#689]
Myself and other members of a cycling group have been mapping much of the town. However, I am not clear what is necessary to show the blue route squares. I've tried various different combinations but they don't show consistently. Is there any guidance on how to achieve this please?

**Comment by Phyks** [comment]
Hi!
I’m not sure what are « blue route squares ». I assume they are the cycling route labels? These are labels (names) from the cycling route relations: https://wiki.openstreetmap.org/wiki/Tag:route%3Dbicycle. 

Happy to help you start tagging these, although please note that you should avoid tagging for the renderer (https://wiki.openstreetmap.org/wiki/Tagging_for_the_renderer). OSM data are used far beyond CyclOSM render and should have a good quality and consistency. 

These cycling route relations should only be used for marked cycling ways (e.g. with road signs), typically for cycling routes mainly for holidays (e.g. vélo route / euro vélo).

**Comment by paultroop** [comment]
Yes, I'm referring to the route names that show up in blue (and other colours) on CyclOSM. I believe that they have been tagged correctly in accordance with OSM standards, but there seems no obvious logic to when they are displayed. There are a number of routes, either extant or proposed, but the sections that appear are often poor / incomplete / unsigned routes, whereas the quality / complete / signed routes do not appear. I'm struggling to understand the underlying logic. I believe they comply with the OSM tags.

**Comment by Phyks** [comment]
Could you point me to a few examples which show / don’t show on CyclOSM so I can have a look?

**Comment by paultroop** [comment]
Sure, I'll send you a message!
